### PR TITLE
init sqlite db for storing meta info

### PR DESCRIPTION
**Pull request by makalaaneesh**

### CODE REVIEW

File: yb-voyager/cmd/common.go [hunk 1]
```diff
@@ -308,6 +318,73 @@ func retrieveMigrationUUID(exportDir string) error {
...
+	return nil
+}
+
+func initMetaDB(path string) error {
+	conn, err := sql.Open("sqlite3", path)
+	if err != nil {
+		return fmt.Errorf("error while opening meta db :%w", err)
+	}
+	cmds := []string{
+		fmt.Sprintf(`CREATE TABLE %s (segment_no INTEGER, size_committed INTEGER);`, QUEUE_SEGMENT_META_TABLE_NAME),
```
Comment (amit-yb) [inline yb-voyager/cmd/common.go:360]: Do we need `IF NOT EXISTS` clause?

File: yb-voyager/cmd/common.go [hunk 2]
```diff
@@ -308,6 +318,73 @@ func retrieveMigrationUUID(exportDir string) error {
...
+		log.Infof("Executed query on meta db - %s", cmd)
+	}
+	return nil
+}
+
+func truncateTablesInMetaDb(exportDir string, tableNames []string) error {
+	conn, err := sql.Open("sqlite3", getMetaDBPath(exportDir))
+	if err != nil {
+		return fmt.Errorf("error while opening meta db :%w", err)
+	}
```
Comment (amit-yb) [inline yb-voyager/cmd/common.go:376]: `conn.Close()`??

File: yb-voyager/cmd/common.go [hunk 3]
```diff
@@ -308,6 +318,73 @@ func retrieveMigrationUUID(exportDir string) error {
 	return nil
 }
 
+func getMetaDBPath(exportDir string) string {
```
Comment (amit-yb) [inline yb-voyager/cmd/common.go:321]: How about creating a new file `metadata-db.go` for all functions related to the meta db?

**Review by amit-yb — APPROVED** [review]
LGTM.

File: yb-voyager/cmd/common.go (hunk 1 above)
Comment (amit-yb) [reply yb-voyager/cmd/common.go:360]: Add a column named `file_path`. It should be segment file path relative to export-dir.

Comment (makalaaneesh) [reply yb-voyager/cmd/common.go]: We only run the DDLs when we init the database for the first time, so `IF NOT EXISTS` is not needed! 

Added `file_path` but how would it help 🤔 queue segment no should be enough, no?

**Pull request merged**